scripts/tests/smoke-test-gui-windows.sh

```diff
@@ -34,25 +37,11 @@ function smoke_test() {
     # Run the smoke test normally
     cargo run --bin "$PACKAGE" -- smoke-test
 
-    # Note the device ID
-    DEVICE_ID_1=$(cat "$DEVICE_ID_PATH")
-
-    # Run the test again and make sure the device ID is not changed
-    cargo run --bin "$PACKAGE" -- smoke-test
-    DEVICE_ID_2=$(cat "$DEVICE_ID_PATH")
-
-    if [ "$DEVICE_ID_1" != "$DEVICE_ID_2" ]
-    then
-        echo "The device ID should not change if the file is intact between runs"
-        exit 1
-    fi
```

Comment:
A lot of this stuff doesn't work now that the IPC service has been split out, and properly running it in CI without just installing it is a little tricky.

In a future PR I should probably install it, but that means fighting the CI more

Reply:
When we get to #5453, that would be a good time to properly smoke-test the IPC service and put all these checks back in.